lib/consumer.ex

```diff
@@ -298,6 +297,20 @@ defmodule GenRMQ.Consumer do
     |> handle_reconnect(state)
   end
 
+  @doc false
+  @impl GenServer
+  def handle_info({:DOWN, _ref, :process, _pid, _reason}, state) do
```

Comment:
Will we get here messages for tasks or supervisor?

Reply:
This should be coming from the task if it failed. I'm thinking about adding a telemetry event in there to capture the error so library users can track processing failures.